sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -120,11 +120,15 @@ def check_max_n_features(est, X, y):
 
     # Test if max_n_features can break tie among feature importance
 
-    for n_features in range(max_n_features):
+    feature_importances = np.array([1, 2, 2, 3, 3, 3, 4, 4, 4, 4])
```

Comment:
If you define this as `[4, 4, 4, 4, 3, 3, 3, 2, 2, 1]` (i.e. reversed) then the below check becomes `assert_array_equal(selected_feature_indices, np.arange(n_features))` which I think is much more elegant and explicit.